niceml/experiments/schemas/expmember.py

```diff
@@ -54,21 +56,39 @@ def __init__(self):
 class FolderMember(ExpMember):
     """This member is a folder containing arbitrary files with specific extensions"""
 
-    def __init__(  # pylint: disable=too-many-arguments
+    def __init__(  # ruff: noqa: PLR0913
         self,
         path: str,
         required: bool,
         description: str,
         min_required_files: int = 0,
         extensions: Optional[List[str]] = None,
     ):
+        """This member is a folder containing arbitrary files with specific extensions"""
+
         super().__init__(
             path=path, required=required, description=description, member_type="folder"
         )
         self.min_required_files = min_required_files
         self.extensions = extensions
 
     def get_docstring(self) -> str:
+        """
+        The get_docstring function is used to generate the docstring for a given
+        :class:`~.FileChecker`. The function takes no arguments and returns a string
```

Comment:
What does ':class:`~.FileChecker`' mean? Can you think of a better wording?

Reply:
I've rewritten the docstring